e107_plugins/social/e_comment.php

```diff
@@ -34,7 +34,7 @@ public function config() // Admin Area Configuration.
 
 		if($engine == 'social::facebook' && empty($this->facebookActive))
 		{
-			e107::getMessage()->addInfo("Facebook comments requires that you have a facebook App ID. See the 'social login' area in admin-preferences to add one.");
+			e107::getMessage()->addInfo(LAN_SOCIAL_WARNING);
```

Comment:
Alredy applied:

`public function config() // Admin Area Configuration.
	{
		$engine = e107::pref('core','comments_engine','e107');

		if($engine == 'social::facebook' && empty($this->facebookActive))
		{
			e107::getMessage()->addInfo(LAN_SOCIAL_WARNING);
		}

		$config = array();
		$config[] = array('name' => "Facebook", 'function'=>'facebook');


		return $config;
	}`